components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/KeyMgtConfigurations.java

```diff
@@ -18,25 +18,24 @@
  */
 
 import org.wso2.carbon.apimgt.core.util.KeyManagerConstants;
-//import org.wso2.carbon.kernel.annotations.Configuration;
-//import org.wso2.carbon.kernel.annotations.Element;
+import org.wso2.carbon.kernel.annotations.Configuration;
+import org.wso2.carbon.kernel.annotations.Element;
 
 /**
  * Class to hold APIM configuration parameters and generate yaml file
- * TODO refactor class when kernal is updated to 5.2.0
  */
-//@Configuration(namespace = "wso2.carbon.kmgt", description = "Key Management Configuration Parameters")
+@Configuration(namespace = "wso2.carbon.kmgt", description = "Key Management Configuration Parameters")
```

Comment:
Since these are configurations for apim (similar to api-manager.xml in C4) can't we use the same namespace